cosmoz-viewinfo.js

```diff
@@ -4,18 +4,17 @@
 
 	window.Cosmoz = window.Cosmoz || {};
 
-	var
-		viewInfoInstances = [],
-		sharedViewInfo = {
-			desktop: false,
-			effects: 10,
-			height: 0,
-			landscape: false,
-			mobile: false,
-			portrait: true,
-			tablet: false,
-			width: 0
-		};
+	let viewInfoInstances = [];
```

Comment:
This is also const, even if the items change, the reference never changes.

Reply:
I realized that after I have pushed the change, fixed.